### PR TITLE
Add the Diverse strings problem

### PR DESCRIPTION
**Pull request by SOrozco7**
Hello!

I just added my solution to this problem. Please, review it.

Thanks
@ChavaO777

### CODE REVIEW

File: A_DiverseStrings.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
+    return true;
+}
+
+int main(){
+
+    int tc;
+    S(tc);
+
```
Comment (Fernando74lr) [inline A_DiverseStrings.cpp:56]: You missed getchar() function

File: A_DiverseStrings.cpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        }
+    }
+
+    return true;
+}
+
+int main(){
+
+    int tc;
+    S(tc);
```
Comment (AleOD) [inline A_DiverseStrings.cpp:54]: You missed the getChar() function

Comment (SOrozco7) [reply A_DiverseStrings.cpp:54]: Thanks, @a01731592. I just fixed that in my last commit.

File: A_DiverseStrings.cpp (hunk 1 above)
Comment (SOrozco7) [reply A_DiverseStrings.cpp:56]: Thanks. I just solved it!

**Review by AleOD — APPROVED** [review]

**Review by Fernando74lr — APPROVED** [review]
This change is approved.

**Pull request merged**